content/tutorials/getusermedia/intro/en/index.html

```diff
@@ -389,118 +387,108 @@ <h2 id="toc-screenshot">Taking screenshots</h2>
   <p><button id="screenshot-button" disabled>Take screenshot</button></p>
 </div>
 
-<h2 id="toc-effects">Applying Effects</h2>
+<h2 id="toc-effects">Apply effects</h2>
 
 <h3 id="toc-effects-css">CSS Filters</h3>
 
-<p>Using <a href="https://dvcs.w3.org/hg/FXTF/raw-file/tip/filters/index.html">CSS Filters</a>, we can apply some gnarly effects to the <code>&lt;video&gt;</code>
+<p>With CSS filters, you can apply some gnarly effects to the <code>&lt;video&gt;</code>
 as it is captured:</p>
-<pre class="prettyprint"><code>
-&lt;video autoplay&gt;&lt;/video&gt;
-&lt;p&gt;&lt;button class="capture-button"&gt;Capture video&lt;/button&gt;
-&lt;p&gt;&lt;button id="cssfilters-apply"&gt;Apply CSS filter&lt;/button&gt;&lt;/p&gt;
-
-&lt;script&gt;
-const captureVideoButton =
-  document.querySelector('#cssfilters .capture-button');
-const cssFiltersButton =
-  document.querySelector('#cssfilters-apply');
-const video =
-  document.querySelector('#cssfilters video');
+<pre class="prettyprint"><code>&lt;video autoplay&gt;&lt;/video&gt;
```

Comment:
HTML code should be left unchanged.

```
&lt;p&gt;&lt;button class="capture-button"&gt;Capture video&lt;/button&gt;
&lt;p&gt;&lt;button id="cssfilters-apply"&gt;Apply CSS filter&lt;/button&gt;&lt;/p&gt;

&lt;script&gt;
```

Reply:
Added back

Reply:
Please leave unchanged

```
<pre class="prettyprint"><code>
&lt;video autoplay&gt;&lt;/video&gt;
```

Not `<pre class="prettyprint"><code>&lt;video autoplay&gt;&lt;/video&gt;`